scripts/Packages/Package_Gullwing__QFP_SOIC_SO/Readme.md

```diff
@@ -0,0 +1,151 @@
+# IPC gullwing generator
+
+This generator uses IPC-7351B equations and fillet definitions to derive a footprint for gullwing style packages. (fillet definitions can be changed by pointing to a different definition file.) The script generates footprints in zero orientation version A (pin 1 at top left corner).
+Pads are generated using rounded rectangle pads as suggested in preliminary releases of IPC-7351C.
+
+Examples of supported packages include: QFN, SOIC, SO.
+
+## Running the script
+
+The script requires python 3.4 or newer. Run it with:
+`python3 ipc_gullwing_generator.py size_definitions/eqfp.yaml` (replace `eqfp.yaml` with the size definition file that contains your part.)
+
+### Optional script parameters
+
+* --global_config: the config file defining how the footprint will look like. (KLC)' (default='../../tools/global_config_files/config_KLCv3.0.yaml')
```

Comment:
The first two descriptions start with 'the' but the last three don't. I suggest removing the 'the' after the colon.

Also, what is the apostrophe doing after `(KLC)`?

Reply:
copied directly from python so the ' is there because it is a string in python.